src/components/data-management/ProjectDetails.jsx

```diff
@@ -589,10 +593,42 @@ const ProjectDetails = ({ width, height }) => {
           Processed Seurat object (.rds)
         </Tooltip>
       </Menu.Item>
-      <Menu.Item disabled>
-        <Tooltip title='Coming soon!' placement='left'>
-          Data Processing settings
-        </Tooltip>
+      <Menu.Item
+        onClick={async () => {
+          const experiment = activeProject.experiments[0];
+          const experimentName = experiments[experiment]?.name
+
+          // load processing configuration
+          await dispatch(loadProcessingSettings(experiment));
+          const config = _.omit(experimentSettings.processing, ['meta']);
+
+          // // TODO: disable button if this is false
+          // const pipelineYetToRun = Object.keys(config).length === 0
+
+          const filteredConfig = Object.entries(config)
+            .map(([step, stepConfig]) =>
+              [
+                step,
+                Object.fromEntries(
+                  activeProject.samples.map((sample) =>
+                  [
+                    samples[sample]?.name,
+                    stepConfig[sample]?.filterSettings ?? stepConfig
+                  ]
+                )
+                )
+              ]
+            )
+
+          // TODO: remove unused method settings
+          // console.log(Object.fromEntries(filteredConfig))
+
+          const string = INI.stringify(Object.fromEntries(filteredConfig))
```

Comment:
Could we move this part into a utils method? `exportJson` or whatever you think is appropriate

Reply:
Right now I am just using `INI.stringify`, but depending on @vickymorrison's feedback, I might switch to something more custom. Probably it would be best to move this whole handler to another file, as it is quite long.

Reply:
Oh, do you mean up to and including the `saveAs`?

Reply:
yes, I guess moving the handler might be nice since it is a pretty generic piece of code that might be reusable in other functionalities in the future.